res/skins/Shade/deck_overview.xml

```diff
@@ -14,9 +14,6 @@
     <PlayPosColor>#00FF00</PlayPosColor>
     <EndOfTrackColor>#EA0000</EndOfTrackColor>
     <PlayedOverlayColor>#60000000</PlayedOverlayColor>
-    <LabelBackgroundColor>#80000000</LabelBackgroundColor>
-    <LabelTextColor>#ffffff</LabelTextColor>
-    <LabelFontSize>9</LabelFontSize>
```

Comment:
This breaks Shade, the label disappears when hovering:
![image](https://user-images.githubusercontent.com/9455094/65204343-6cdb5e00-da52-11e9-8a0d-360af13e4d2e.png)
9 is the maximum font size that fits

Reply:
If you want to increase the vertical height of the overview to allow a bigger text size, that would be fine with me.

Reply:
Not for me: 
![grafik](https://user-images.githubusercontent.com/1777442/65186801-b0da5d00-da6a-11e9-89dc-ac434f5bf76e.png)

Does this overlap also happen in a unscaled version? ... Or is it a different font issue?

Reply:
This also happens for me with 1x scaling:
![image](https://user-images.githubusercontent.com/9455094/65248678-2c163000-dab8-11e9-8445-3836a77b0d8b.png)

Reply:
Ok, so I guess we need to set a font as well.

Reply:
You're right, I still had automatic scaling enabled so that was at 2x. Here it is at 1x:
![image](https://user-images.githubusercontent.com/9455094/65267114-fe41e300-dad9-11e9-9dd6-da0c6cc24c9a.png)

Reply:
Got it. See the latest commit. Your font seems to have higher charters for Î or such. But this is not relevant here, so we can use the pixel high, which only encounters the height of "Hg"

Reply:
Why should we assume that the user isn't using tall characters like Î? The label can be any text the user wants.

Reply:
The user can still use Î, depending to the font it may now hit or exceed the background. This is OK for me since the majority of characters are looking well.

Reply:
I don't think this is a good idea. The area of the background should cover any possible character.